layouts/partials/main-menu-mobile.html

```diff
@@ -2,7 +2,9 @@
   <ul>
     {{ $currentPage := . }}
     {{ range .Site.Menus.main }}
-    <li class="menu-item-{{ .Name | lower }}{{ if $currentPage.IsMenuCurrent "main" . }} active{{ end }}">
+    {{ $active := or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }}
+    {{ $active = or $active (eq (.URL | relLangURL) ($currentPage.RelPermalink | relLangURL)) }}
```

Comment:
I just noticed a slight inconsistency in the use of `$currentPage`. We might want to address that by changing to either one of the following:

```suggestion
    {{ $active := or (.IsMenuCurrent "main" .) (.HasMenuCurrent "main" .) }}
    {{ $active = or $active (eq (.URL | relLangURL) (.RelPermalink | relLangURL)) }}
```

```suggestion
    {{ $active := or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }}
    {{ $active = or $active (eq ($currentPage.URL | relLangURL) ($currentPage.RelPermalink | relLangURL)) }}
```

Reply:
$currentPage is needed. It holds the Page object. We declare $currentPage outside of the range because the dot context inside the range is a MenuEntry. We need to compare the Page object to the MenuEntry object. 


The below wont work. because .IsMenuCurrent would be operating on the MenuEntry object inside the range and this method only works on a Page
```
{{ range .Site.Menus.main }}
{{ .IsMenuCurrent "main" . }} 
{{. end  }}
```

The below wont work because .RelPermalink only exists on Page object, and here the `.` is a MenuEntry
```
{{ range .Site.Menus.main }}
    {{ $active = or $active (eq (.URL | relLangURL) (.RelPermalink | relLangURL)) }}
{{. end  }}
```

The existing code does seem correct to me
```
{{ $active := or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }}
{{ $active = or $active (eq (.URL | relLangURL) ($currentPage.RelPermalink | relLangURL)) }}
```

Reply:
Oh, yeah, you're absolutely right – I didn't think of that when writing my last comment. I'll mark as resolved.

Anything else to get this ready for merging?